src/actions/base.ts

```diff
@@ -124,9 +124,8 @@ export class BaseAction {
       return false;
     }
     if (
-      vimState.recordedState.getCurrentCommandWithoutCountPrefix().length - keysPressed.length >
-        0 &&
-      this.mustBeFirstKey
+      this.mustBeFirstKey &&
```

Comment:
Should take advantage of short-circuit evaluation to avoid work, only 3 commands have `mustBeFirstKey` set.